src/main/java/org/omnifaces/resourcehandler/WebAppManifest.java

```diff
@@ -198,12 +198,13 @@ public Display getDisplay() {
     }
 
     /**
-     * Returns the default orientation mode of your web application. The default implementaiton returns {@link Orientation#ANY}.
+     * Returns the default orientation mode of your web application. The default implementation returns null to let the device handle the orientation
+     * based on the combination of device orientation and user auto-orientation setting.
```

Comment:
- use `{@code null}` instead of plain text null
- javadoc should tell since which version the return value has changed and what value it was before that version